src/addrman.h

```diff
@@ -264,6 +264,17 @@ friend class CAddrManTest;
     void SetServices_(const CService &addr, ServiceFlags nServices) EXCLUSIVE_LOCKS_REQUIRED(cs);
 
 public:
+    //! Serialization versions.
+    enum class Format : uint8_t {
+        V0_HISTORICAL = 0, //!< historic format, before commit e6b343d88
+        V1_DETERMINISTIC = 1, //!< for pre-asmap files
+        V2_ASMAP = 2, //!< for files including asmap version
+        V3_BIP155 = 3, //!< same as V2 plus addresses are in BIP155 format
```

Comment:
pico-nit, as `clang-format` suggests, and s/V2/V2_ASMAP/ in comment:
```suggestion
        V0_HISTORICAL = 0,    //!< historic format, before commit e6b343d88
        V1_DETERMINISTIC = 1, //!< for pre-asmap files
        V2_ASMAP = 2,         //!< for files including asmap version
        V3_BIP155 = 3,        //!< same as V2_ASMAP plus addresses are in BIP155 format
```

Reply:
Done